### PR TITLE
Give users ability to re-order columns

### PR DESCRIPTION
**Pull request by aclark02-arcus**
I don't think I ever opened an issue for this one here, but this is probably the most requested feature to `clinsight` for us. Kind of related to #96 because it reduces reliance on exporting to excel, when you can control the variable order in the app. I was too lazy to record a `gif`, but give it a run and you can now drag and drop columns for any table.

The only known issue with this extension is listed here: https://github.com/rstudio/DT/issues/250

### CODE REVIEW

**Comment by LDSamson** [comment]
Looks okay to me. Probably nice to give users some flexibility here. 

The preference will of course not be saved. Does everyone in your team have their own column preferences? If so: the column order is usually set based on the order of the variables in the `metadata.xlsx` file.

File: R/fct_data_helpers.R [hunk 1]
```diff
@@ -542,10 +542,10 @@ datatable_custom <- function(
     rename_vars = NULL, 
     title = NULL, 
     selection = "single",
-    extensions = "Scroller",
+    extensions = c("Scroller", "ColReorder"),
     plugins = "scrollResize",
     dom = "fti",
-    options = list(),
+    options = list(colReorder = TRUE),
     ...
```
Comment (LDSamson) [inline R/fct_data_helpers.R:549]: @aclark02-arcus I think it is probably better to add this to the list of modifyable defaults and add it to the function's description. @jthompson-arcus what do you think? In addition, can you also update the package documentation and include it in the PR?

**Review by LDSamson — CHANGES_REQUESTED** [review]

File: R/fct_data_helpers.R (hunk 1 above)
Comment (jthompson-arcus) [reply R/fct_data_helpers.R:549]: Yes, with how this function is currently setup, default options should be included in `default_opts`, not the options argument.

**Review by LDSamson — CHANGES_REQUESTED** [review]
Can you review and fix the test errors? otherwise I think it looks okay.

The tests capture the datatable output in their snapshot, I think that is why it is failing.

**Comment by aclark02-arcus** [comment]
Hi @LDSamson, got the coveted ✅

**Review by LDSamson — APPROVED** [review]
Looks good to me

**Pull request merged**